src/pages/components/tags.mdx

```diff
@@ -0,0 +1,128 @@
+---
+title: Tags
+description: >
+  A Tag is an interactive label used to categorize content.
+package: '@zendeskgarden/react-tags'
+components:
+  - tags/src/elements/Tag.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To categorize by dimension, attribute, or property</li>
```

Comment:
- To add metadata to an element such as category, attribute, or property
- To communicate status
- To represent parameters of a filter